test/tools/fuzzTesting/fuzzHelper.h

```diff
@@ -47,14 +48,20 @@ class RandomCode
 	static RandomCodeBase& get()
 	{
 		static RandomCode instance;
-		return instance.generator;
+		return *instance.generator;
 	}
 	RandomCode(RandomCode const&) = delete;
 	void operator=(RandomCode const&) = delete;
+	~RandomCode() { delete generator; }
```

Comment:
No kidding. This is no go. Use `std::unique_ptr`. But probably not needed at all according to my other comments.